WebHostLib/templates/userContent.html

```diff
@@ -33,7 +33,18 @@ <h2>Your Rooms</h2>
                             <tr>
                                 <td><a href="{{ url_for("view_seed", seed=room.seed.id) }}">{{ room.seed.id|suuid }}</a></td>
                                 <td><a href="{{ url_for("host_room", room=room.id) }}">{{ room.id|suuid }}</a></td>
-                                <td>{{ room.seed.slots|length }}</td>
+                                <td>
+                                    {% set title = [] %}
+
+                                    {% for slot in room.seed.slots %}
```

Comment:
I think if you add this then they should get sorted properly.

```suggestion
                                    {% for slot in room.seed.slots|list|sort(attribute="player_id") %}
```

Reply:
The `sort` after in the `title` attribute after this block had to be removed. Otherwise it would have overwritten the sort again.